examples/hata_basic_queue/bot.py

```diff
@@ -261,11 +256,11 @@ async def user_voice_update(client, event, _old):
     )
     sys.exit(0)
 
+    # this is unreachable...
```

Comment:
what should i do about this?

Reply:
Personally, I would comment out all this code and add a note saying why its commented out

Reply:
well it does sys.exit(0) before so i don't understand why it's even there...

Reply:
Line above explains why the exit is there

Reply:
ah I didn't read that